### DISCUSSION TITLE
Revert sampler back to stock firmware

### DISCUSSION THREAD

**Discussion by Bornin92** [bangcorrupt/hacktribe#138] [Q&A] [answered]
I've tried to revert my e2s back to stock firmware with no luck. I wanted to have the option to jump back and forth from hacktribe & stock. Any idea how to do it ?

**Comment by bangcorrupt** [comment, marked as answer]
Reverting to the stock sampler firmware should work the same as any normal update, can you be more specific about what happened?  

~Swapping between hacktribe and sampler on one SD card is not currently possible as they use the same file path.~

**Reply by Bornin92** [reply]
I got the stock firmware and did the update it still started up as hacktribe

**Reply by bangcorrupt** [reply]
If the update ran without errors and it booted into hacktribe, the VSB file installed was hacktribe.  Check you are putting the file at the correct path for sampler, delete any existing file first.

~`KORG/'electribe sampler'/System/SYSTEM.VSB`~

`KORG/hacktribe/System/SYSTEM.VSB`

**Reply by Bornin92** [reply]
Okay I'll give that another shot today and report back !

**Reply by bangcorrupt** [reply]
Since 3d31b7c, hacktribe will look for files under `KORG/hacktribe`.  This will allow swapping back and forth.
